hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/replication/TestMeasuredReplicator.java

```diff
@@ -119,4 +119,16 @@ public void testSuccessTimeFailureExcluded() {
     Assert.assertEquals(0, measuredReplicator.getSuccessTime().value());
   }
 
+  @Test
+  public void testReplicationQueueTimeMetrics() {
+    ReplicationTask task = new ReplicationTask(100L, new ArrayList<>());
+    try {
+      Thread.sleep(1000L);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
```

Comment:
Instead of adding sleep, we can override `getQueued()` in `task` to return a past value.  This saves time and reduces randomness.

(Note: needs extra import)

```suggestion
    final Instant queued = Instant.now().minus(2, ChronoUnit.MINUTES);
    ReplicationTask task = new ReplicationTask(100L, new ArrayList<>()) {
      @Override
      public Instant getQueued() {
        return queued;
      }
    };
```

Reply:
Thanks for the advice. Updated the commit, please have a check.